tnscope_eval.snakefile

```diff
@@ -42,7 +42,7 @@ configfilepath = f"{ROOT_DIR}/configs/config_hg38.json"
 
 
 pipeconfig = helpers.read_config(configfilepath)
-clusterconf = helpers.read_clusterconf()
+clusterconf = helpers.read_config(f"{ROOT_DIR}/configs/cluster.yaml")
```

Comment:
I am not a fan of specifying the path to the config twice. You have already specified it in pipeline.snakefile, can we re-use or simplify, specify it in a global variable? 
Maybe this is easiest way though, given how the code is setup, so maybe we should just be happy that it is simplified enough with the use of root_dir?

Reply:
I think this is its own snakefile made just for evaluation of TNscope? This file is not used in normal execution but I wanted to make sure it would still work with the removal of the read_clusterconf(). There are quite a few scripts/snakefiles that are no longer used or were created for a single run, that are just hanging around in the repo.